airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py

```diff
@@ -88,14 +88,10 @@ def secret_property_names_fixture():
 DATETIME_PATTERN = "^[0-9]{4}-[0-9]{2}-[0-9]{2}(T[0-9]{2}:[0-9]{2}:[0-9]{2})?$"
 
 
-@pytest.mark.default_timeout(30)
+@pytest.mark.default_timeout(60 * 5)
```

Comment:
I'm curious why we need 5 minutes to run spec tests

Reply:
This can be reverted, was for debugging, thanks you for noticing it.